read-the-docs/2/requirements.txt

```diff
@@ -1,6 +1,6 @@
-sphinx==7.4.7
-sphinx_rtd_theme==2.0.0
+sphinx==8.1.3
+sphinx_rtd_theme==3.0.1
 sphinx-autobuild==2024.10.3
 sphinxcontrib.httpdomain==1.8.1
-m2r2==0.3.3.post2
```

Comment:
this extensions prevents from upgrading sphinx. this one is a bad maintained/unmaintained extension.